environment.yml

```
@@ -68,30 +68,29 @@ dependencies:
 
   # Visualization packages
   - bokeh~=3.2.2
-  - cartopy~=0.22.0 # geospatial plotting with matplotlib
+  - cartopy~=0.23.0 # geospatial plotting with matplotlib
   - geemap~=0.32
   - geoviews~=1.10.1
-  - hvplot~=0.8.4
-  - ipyleaflet~=0.17.3
+  - hvplot~=0.10.0
+  - ipyleaflet~=0.19.2
   - ipympl~=0.9.3 # This enables matplotlib interaction with jupyter widgets
-  - leafmap~=0.31
+  - leafmap~=0.36
   - lonboard~=0.9
-  - matplotlib~=3.8.0
+  - matplotlib~=3.9.0
   - plotly~=5.18.0
-  - seaborn==0.12.1 # statistical plotting with matplotlib
+  - seaborn~=0.13.2 # statistical plotting with matplotlib
 
   # Geospatial data packages
-  - geopandas~=0.13.2
+  - geopandas>=1.0
```

Use geopandas pre-1.0 to satisfy snowexsql pinning.

```suggestion
  - geopandas>=0.14.4
```